x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/alert_actions.tsx

```diff
@@ -165,7 +161,7 @@ export function AlertActions({
       key="viewAlertDetailsFlyout"
       onClick={() => {
         closeActionsPopover();
-        setFlyoutAlert({ fields: alertDoc });
+        setFlyoutAlert(alert._id);
```

Comment:
The 'Mute' / 'Unmute' action isn't available for the ML anomaly detection rule from the ML alerts table, which I gather from @XavierM is because we don't `alertInstanceId`. 

@darnautov should we be populating `alertInstanceId`, which would allow us to mute alerts from the table? Or is this not needed for our rule as we only ever have 1 active instance per rule?

<img width="1073" alt="image" src="https://github.com/elastic/kibana/assets/7405507/2d17e466-d341-4d87-9148-79bbf972cbee">

Note I can mute the alert in the existing alerts table in the rule page:

<img width="1363" alt="image" src="https://github.com/elastic/kibana/assets/7405507/2ea0526a-1f9a-49f4-8698-d6738b5990b9">

Reply:
Yes, we only support 1 alert instance per rule atm. We can populate the `alertInstanceId` field but it's always equal the anomaly detection job ID.

Reply:
If this means we can mute the alert from the alerts table inside ML, it's worth populating the `alertInstanceId` I think. What do you think @darnautov @XavierM ?

Reply:
Sure @darnautov! 🙂